src/python/pants/option/options.py

```diff
@@ -80,7 +79,7 @@ def complete_scopes(cls, scope_infos: Iterable[ScopeInfo]) -> Set[ScopeInfo]:
 
         Also adds any deprecated scopes.
         """
-        ret = {GlobalOptionsRegistrar.get_scope_info()}
```

Comment:
This would now cause a circular dependency, but in practice wasn't necessary as only a handful of tests were relying on this, and they now pass in the global ScopeInfo where necessary (the non-test callers were already doing so).